content/login/login.html

```diff
@@ -77,19 +77,19 @@ <h1 id="headerTitle">Serify</h1>
         <h1 id="howText">How does it work?</h1>
```

Comment:
I think we had better keep it for now... because the page looks really empty now and people might want to know what the flowchart is before reading the text in it. Anyway we will have a revamped homepage soon so it won't be bugging you for too long ha